spaceMissionManager-persistance/src/main/java/cz/muni/fi/Entity/CraftComponent.java

```diff
@@ -1,12 +1,75 @@
 package cz.muni.fi.Entity;
 
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
 import java.time.ZonedDateTime;
 
+@Entity
 public class CraftComponent {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
 
+	@Column
```

Comment:
column not necessary same as notNull annotation as this is not a nullable object